wp-includes/functions.php

```diff
@@ -5736,19 +5736,36 @@ function wp_validate_boolean( $var ) {
 /**
  * Delete a file
  *
- * @since 4.2.0
+ * @since 4.9.2
  *
- * @param string $file The path to the file to delete.
+ * @param string $file                The path to the file to delete.
+ * @param array  $directories_unblock Are directory to unblocked. They are ROOT, WP_CONTENT_DIR, WP_ADMIN_DIR
+ *                                   and WP_INCLUDE_DIR.
+ * @return bool Whether the param is invalidated.
  */
-function wp_delete_file( $file ) {
+function wp_delete_file( $file, $directories_unblock = array() ) {
+
+	// Default Filters the path of the file to delete.
+	$path_blocked['ROOT']           = rtrim( ABSPATH, '/' );
```

Comment:
You need to initialize the array with a `$path_blocked = array()` before this line or create the array like so:
```php
$path_blocked = array(
    'ROOT'           => rtrim( ABSPATH, '/' ),
    'WP_CONTENT_DIR' => WP_CONTENT_DIR,
    ...
);
```
Otherwise, PHP will throw error notices.

Reply:
Second option is better for me. Done.